Runner.php

```diff
@@ -15,34 +16,28 @@ public function __construct($config, $server)
 
     public function queueDirectory($dir, $prefix = '')
     {
-        if(isset($this->config['include_paths'])) {
-            $this->queueWithIncludePaths();
-        } else {
-            $this->queuePaths($dir, $prefix, $this->config['exclude_paths']);
```

Comment:
So I think there's a non-trivial matrix of scenarios here, and again we need to be careful about cc.yml exclude-paths vs config.js exclude-paths when discussing this.  I have to step away for a minute, but lets sync up in chat about it when I'm back.